bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoHandler.java

```diff
@@ -480,20 +475,32 @@ public String getWemoURL(String actionService) {
                 }
             }
             wemoURL = "http://" + host + ":" + port + "/upnp/control/" + actionService + "1";
+            logger.trace("WeMo url {}", wemoURL);
             return wemoURL;
         }
         return wemoURL;
     }
 
     public boolean servicePing(String host, int port) throws IOException {
```

Comment:
So IOException this would never get thrown now right?

Reply:
Correct. It's caught and used to set the flag to move on to the next port.

Reply:
You should remove IOException from the throws clause then.

Reply:
Whoops sorry.